### PR TITLE
Make MJ Maintainer, and add Release Notes to General Overview.

### LINKED ISSUE #1357: Further docs review
The release notes should be visible in the main screen list of documents on the general overview page:

https://docs.pymedphys.com/general/index.html

### PR DESCRIPTION
**Pull request by Matthew-Jennings**
Closes #1357

### CODE REVIEW

File: README.rst
Comment (Matthew-Jennings) [inline README.rst:111]: Is this okay, @SimonBiggs & @sjswerdloff?

I'm still in a position where I need to carefully manage my workload, and this impacts my ability to review. But I'm keen to be pretty active again and help PyMedPhys grow.

Comment (SimonBiggs) [reply README.rst:111]: I guess it's probably worth sorting out and making expectations clear before jumping in. What are your thoughts about the following level of expectations?

- When a PR only involves "site-specific" or "prototyping" code no review needed, and best not to even flag someone.
- When a PR involves only "experimental" code a review is helpful, but not required. Please still mark someone relevant to review.
- When a PR involves standard library code a review is required before merge.
- When a review is requested from a maintainer, aim to have some form of response within 24 hours taking into consideration the need for days off (and weeks off). For example, my 'day off' is Sunday, so that day isn't included in that expectation. @sjswerdloff's days off are Friday and Saturday. Also, I will be going on leave in two weeks for about two weeks, and I will be completely unresponsive during that time.
- A response does not imply a review, or even a look at the code. It can simply be a "I can't deal with this right now, can someone else review", or "This is really important to me, I can't review it right now, can I look at it next week?".
- In the cases where for whatever reason maintainers are unresponsive experimental code can be merged. Standard library code can be merged but it at least needs a review from a contributor.

Comment (Matthew-Jennings) [reply README.rst:111]: That sounds pretty fair. Depending on how often I'm specifically asked for, 24 hours might be a little ambitious _for me_, since I suspect I spend a lot less time at my PC than you, and even myself before 2020. My life is pretty full and active or non-coding stuff, but it's also pretty changeable at the moment.

I think one key thing (notably missing from your list!) that I'm wary of is any expectation of max time taken to complete a review. For me, this can especially be the case for more backend/structural stuff where your expertise dwarfs mine. It's not so easy for me to provide a useful review without devoting considerable time to learning and explanation. Many times, this is actually really good! I've learned quite a lot and often really enjoy this stuff. But some may prove beyond my capacity (workload and perhaps even interest-wise sometimes). And I suspect that some of these structural changes, if they're large and frequent, may introduce a high cognitive load for possibly little benefit from (my) review.

Why don't we just see how we go? I'm more than happy to remain listed as Maintainer Emeritus. You and Stuart can decide on what level of activity should be met to be listed as a Maintainer, and whether I meet it, at a later date after I've dived in some more and reviewed some more. After a while, we'll all have a better idea of what's in my capacity. :)

Comment (SimonBiggs) [reply README.rst:111]: > I think one key thing (notably missing from your list!)

A little on purpose actually. No expected time-frame for total review time. The key component is responsiveness both on the initial review and through the review process. If a review is going to take a significant amount of time, as long as that is flagged up front, that's just fine.

There is another expectation that I need to be able to flag:

* a maintainer at any time, without needing to justify themselves can request that a PR that covers the standard library code be split up into smaller more easily reviewable pieces.

Comment (SimonBiggs) [reply README.rst:111]: > Why don't we just see how we go?

Let's let it sit until Monday and give @sjswerdloff a chance to chime in 🙂, given I think the decision of becoming a maintainer should be decided by all current maintainers (not just one of them). Also... the expectations of being a maintainer should be decided by all maintainers too 🙂.

Comment (Matthew-Jennings) [reply README.rst:111]: Sounds good 😊

Comment (sjswerdloff) [reply README.rst:111]: I think the most important things are:
Ethical behaviour
Good faith effort
@Matthew-Jennings  is a known quantity, and it's clear he is being careful about setting expectations and what he will commit to (no pun intended).

with respect to knowledge for performing a review:
Having multiple reviewers available means that there can be overlap in what gets a "quality" review.
Some stuff, I just don't really understand (at least to any depth), e.g. machine learning,  but that's OK.  I make noise about the lack of docstring and type hints.
Some stuff, I am a so called "expert" (e.g. DICOM), so I tend to analyse that deeply.
@Matthew-Jennings has expertise in other things, right?
So if the two of us are reviewing the best we can, the combined review is much better than either alone.

with respect to splitting up large PRs.
Preach.
For that alone, I would "sign him up".
I got hit with that in my first try.
And it was a very important lesson and experience.
Having two of us to hold @SimonBiggs feet to the fire on his own (absolutely vital) rules:  Precious :-)

Welcome to being a Maintainer @Matthew-Jennings

**Review by sjswerdloff — APPROVED** [review]

**Comment by sjswerdloff** [comment]
But I guess my review delay resulted in a setup.py conflict.
Sorry...  Please sort that out (it should be pretty mechanical)

**Comment by sjswerdloff** [comment]
I think the conflict is a perfect example of the value of splitting up a PR

File: README.rst
Comment (SimonBiggs) [reply README.rst:111]: > Having two of us to hold @SimonBiggs feet to the fire on his own (absolutely vital) rules: Precious :-)

:rofl:

:fire: :footprints: :fire: ... I can already feel it.

**Comment by SimonBiggs** [comment]
> I think the conflict is a perfect example of the value of splitting up a PR

Unfortunately I suspect that would have happened anyway. The conflict is due to `setup.py` including a copy of `README.rst` within it all on one line. ... Essentially, as it stands, if two PRs make a change to the README it will currently always cause a `setup.py` conflict.

...I should be able to fix that with the PyMedPhys bot though :) See https://github.com/pymedphys/pymedphys/issues/1366

**Review by SimonBiggs — APPROVED** [review]
Has my approval :tada: 

... I think celebratory drinks are in order ... 

:beer: :beers: :beer:

---

Feel free to merge once you're happy with the PR :slightly_smiling_face:.

**Comment by Matthew-Jennings** [comment]
😂 🍺 🎆

File: README.rst
Comment (Matthew-Jennings) [reply README.rst:111]: Thanks very much, @sjswerdloff !

**Comment by Matthew-Jennings** [comment]
@SimonBiggs, @sjswerdloff, I'm sorry, I merged this when it said "All checks have passed" at the bottom of the PR Conversation and then later noticed that there's now a failure (perhaps of a test not marked as required for PR merging)?

Should I revert the merge?

**Comment by SimonBiggs** [comment]
All good. Don't stress about that OnlineApp one. Only tests that are marked as "required" are needed for a merge.

**Pull request merged**
